clients/web/src/rest.js

```diff
@@ -171,6 +172,12 @@ let restRequest = function (opts) {
         throw new Error('restRequest requires a "url" argument');
     }
     args.url = `${getApiRoot()}${args.url.substring(0, 1) === '/' ? '' : '/'}${args.url}`;
+    // jQuery's ajax method prefers `method` over `type`.  Since we specify a
+    // default `method`, if the caller supplies a `type`, use it instead.
+    if (args.type) {
```

Comment:
`$.ajax` also has a default of `method: 'GET'`. Instead of this block, could we just remove line 108, then allow `$.ajax` to overwrite the default with either `type` or `method`, if they're supplied?